tests/conftest.py

```diff
@@ -1287,3 +1287,36 @@ def duts_minigraph_facts(duthosts, tbinfo):
         }
     """
     return duthosts.get_extended_minigraph_facts(tbinfo)
+
+def collect_db_dump_on_duts(request, duthosts):
+    '''
+        When test failed, teardown of this fixture will dump all the DB and collect to the test servers
+    '''
+    if request.node.rep_call.failed:
+        db_dump_base = "/tmp/db_dump"
+        db_dump_path = os.path.join(db_dump_base, request.module.__name__, request.node.name)
+        db_dump_tarfile = "{}.tar.gz".format(db_dump_base)
+
+        # Collect DB config
+        dbs = {}
+        result = duthosts[0].shell("cat /var/run/redis/sonic-db/database_config.json")
+        logger.debug("db_config: {}".format(result['stdout']))
```

Comment:
I think this log is unnecessary. The previous line should already have logs of showing the file content on DUT.

Reply:
Got it, Thanks!